postprocessing/comstockpostproc/naming_mixin.py

```diff
@@ -916,7 +961,7 @@ def col_name_to_savings(self, col_name, new_units=None):
         elif "_bill.." in converted_col_name:
             converted_col_name = converted_col_name.replace('_bill..', '_bill_savings..')
         elif "peak_" in converted_col_name:
-            converted_col_name = converted_col_name.replace('peak_', 'peak_savings_')
+            converted_col_name = converted_col_name.replace('_daily_peak_', '_daily_peak_savings_')
```

Comment:
hm if you are converting `_daily_peak_` to something else, wouldn't it be safer to check `_daily_peak_` is in `converted_col_name` rather than using `peak_` ? looking at the `elif` right above using `..` at the end makes me think that could be more strict for filtering the column names.. but I also know you went through this so will leave it to your decision. I think I was thinking towards like, someone adds a variable name including `peak_` and postprocessing collects those columns to calculate additional things even though it was not intended.